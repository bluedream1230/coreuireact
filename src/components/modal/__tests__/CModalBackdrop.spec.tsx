import * as React from 'react'
import { render /* ,screen */ /* ,fireEvent */ /* ,waitFor */ } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { CModalBackdrop } from '../../../index'

test('loads and displays CModalBackdrop component', async () => {
  const { container } = render(<CModalBackdrop>Test</CModalBackdrop>)
  expect(container).toMatchSnapshot()
})

test('CModalBackdrop customize', async () => {
  const { container } = render(<CModalBackdrop className="bazinga">Test</CModalBackdrop>)
  expect(container).toMatchSnapshot()
  expect(container.firstChild).toHaveClass('bazinga')
  expect(container.firstChild).toHaveClass('modal-backdrop')
})
