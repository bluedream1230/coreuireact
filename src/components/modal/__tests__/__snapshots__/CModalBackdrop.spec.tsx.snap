// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CModalBackdrop customize 1`] = `
<div>
  <div
    class="modal-backdrop bazinga"
  >
    Test
  </div>
</div>
`;

exports[`loads and displays CModalBackdrop component 1`] = `
<div>
  <div
    class="modal-backdrop"
  >
    Test
  </div>
</div>
`;
